source/pages/albedo.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    ax.set_ylabel('PSM3 Albedo')
</pre><p>I guess the y-label should just be <code>'Albedo'</code> to be consistent with the remaining plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>